rpm_spec/subpackages/manageiq-gemset

```diff
@@ -46,6 +46,30 @@ Requires: network-scripts
 %description gemset
 %{product_summary} Gemset
 
+%post gemset
+# Remove any old systemd unit files
+provider_worker_types="event_catcher metrics_collector refresh operations"
+provider_types="amazon_cloud_manager ansible_tower_automation_manager azure_cloud_manager autosde_storage_manager azure_stack_cloud_manager azure_stack_network_manager foreman_configuration_manager google_cloud_manager google_network_manager ibm_cloud_vpc_cloud_manager ibm_cloud_power_virtual_servers_cloud_manager ibm_terraform_configuration_manager kubernetes_container_manager kubernetes_monitoring_manager kubevirt_infra_manager lenovo_physical_infra_manager microsoft_infra_manager nsxt_network_manager nuage_network_manager openshift_container_manager openshift_monitoring_manager openstack_cloud_manager openstack_infra_manager openstack_network_manager openstack_storage_manager_cinder_manager amazon_storage_manager_s3 redhat_infra_manager redhat_network_manager redfish_physical_infra_manager vmware_cloud_manager vmware_infra_manager"
+
+files=""
+for provider_type in ${provider_types}
+do
+  for provider_worker_type in ${provider_worker_types}
+  do
+    files+="${provider_type}_${provider_worker_type} "
+  done
+done
+files+="amazon_agent_coordinator"
```

Comment:
This one doesn't fit the rest of the model of "#{manager_type}_#{worker_type}" and I didn't want to cross just "amazon" with all the other workers.  Seemed better to simply leave it until the end